scheduler/src/cook/mesos/share.clj

```diff
@@ -60,14 +60,14 @@
    \"default\" user. If there is NO \"default\" value for a specific type,
    return Double.MAX_VALUE."
   [db user]
-  (->> (util/get-all-resource-types db)
+  (->> [:cpus :mem]
```

Comment:
I'm working on a new branch that includes the scheduler logic as well as
API. I've been developing them together as I discover how to structure the
code. The aforementioned tests have all been reconciled, and there will be
full support for GPU shares.

On Mon, Sep 12, 2016 at 4:17 PM wyegelwel <notifications@github.com> wrote:

> In scheduler/src/cook/mesos/share.clj
> <https://github.com/twosigma/Cook/pull/177#discussion_r78456641>:
>
> > @@ -60,14 +60,14 @@
> >     \"default\" user. If there is NO \"default\" value for a specific type,
> >     return Double.MAX_VALUE."
> >    [db user]
> > -  (->> (util/get-all-resource-types db)
> > +  (->> [:cpus :mem]
>
> Which tests broke? I would prefer not to have to do this but on the other
> hand, don't want you going down a rabbit hole
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/twosigma/Cook/pull/177/files/951550ae3ca37de5e3e028993c3fe0da120bdfec#r78456641>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAlwC21HhDDfDraenP1cJlb_z1QnuSb0ks5qpcFtgaJpZM4J3Wwe>
> .
>